FsAutoComplete/CommandInput.fs

```diff
@@ -138,10 +224,11 @@ module CommandInput =
     | null -> Quit
     | input ->
       let reader = Parsing.createForwardStringReader input 0
-      let cmds = compilerlocation <|> helptext <|> declarations <|> lint <|> parse <|> project <|> completionTipOrDecl <|> quit <|> colorizations <|> error 
-      let cmd = reader |> Parsing.getFirst cmds
```

Comment:
Can do!  This was mostly to help with my local testing. Now that the commands are working I can revert this,